### PR TITLE
updating formatting

### PR DESCRIPTION
**Pull request by kalexand-rh**
Follow up to https://github.com/openshift/openshift-docs/pull/12889 to update some code block formatting.

@rhatdan, does this sound ok to you?

### CODE REVIEW

File: creating_images/s2i_testing.adoc [hunk 1]
```diff
@@ -93,60 +91,65 @@ test:
...
 The *_test_* script assumes you have already built the image you want to
-test. If required, first build the S2I image using:
+test. If required, first build the S2I image. Run one of the following commands:
 
-****
-`$ podman build -t _<BUILDER_IMAGE_NAME>_`
-or
-`$ docker build -t _<BUILDER_IMAGE_NAME>_`
-****
+* If you use CRI-O, run the following command:
```
Comment (rhatdan) [inline creating_images/s2i_testing.adoc]: s/CRI-O/podman/g

File: creating_images/s2i_testing.adoc [hunk 2]
```diff
@@ -93,60 +91,65 @@ test:
...
 +
-====
-
-****
-`$ podman run _<BUILDER_IMAGE_NAME>_ .`
-or
-`$ docker run _<BUILDER_IMAGE_NAME>_ .`
-****
-====
+* If you use CRI-O, run the following command:
```
Comment (rhatdan) [inline creating_images/s2i_testing.adoc]: s/CRI-O/podman/g

File: creating_images/s2i_testing.adoc [hunk 3]
```diff
@@ -93,60 +91,65 @@ test:
...
 +
-====
-
-****
-`$ podman run _<OUTPUT_APPLICATION_IMAGE_NAME>_`
-or
-`$ docker run _<OUTPUT_APPLICATION_IMAGE_NAME>_`
-****
-====
+* If you use CRI-O, run the following command:
```
Comment (rhatdan) [inline creating_images/s2i_testing.adoc]: s/CRI-O/podman/g

**Comment by kalexand-rh** [comment]
@rhatdan, did I interpret your comments correctly?

**Pull request merged**